fastlane_core/lib/fastlane_core/update_checker/changelog.rb

```diff
@@ -21,6 +21,11 @@ def show_changes(gem_name, current_version)
       end
 
       def releases(gem_name)
+        # We have to follow redirects, since some repos were moved
+        # away into a separate org
+        # Taken from https://github.com/excon/excon/issues/115#issuecomment-40647211
+        Excon.defaults[:middlewares] << Excon::Middleware::RedirectFollower
```

Comment:
For global settings like this, I think it would be better if we have an earlier, more central place where we configure them.

I think it gets difficult to reason about the behavior of Excon if its behavior can vary depending on whether it gets used before or after this method gets called.

Rails has the concept of "initializers" where you can set up global settings for your app. Would it make sense for our tools/projects to have a similar place or concept for setting up global configuration? Do we already have a good place?